Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.yml

```diff
@@ -244,7 +244,7 @@ script:
     name: abuseipdb-get-blacklist
     outputs:
     - contextPath: AbuseIPDB.Blacklist
-      description: A list of blacklisted IP addresses.
+      description: A list of IP addresses on a block list.
```

Comment:
are we ok with mixed terminology in this case?
the path is called `blacklist` which we cannot change while the description is using `block list`.